### PR TITLE
add api proxy

### PR DESCRIPTION
**Pull request by PlexPt**

### CODE REVIEW

File: agent/env.example.json [hunk 1]
```diff
@@ -1,3 +1,5 @@
 {
-  "OPENAI_API_KEY": "Your Key Here"
-}
\ No newline at end of file
+  "OPENAI_API_KEY": "Your Key Here",
+  "OPENAI_PROXY_URL": "http://api.openai.com"
```
Comment (keldenl) [inline agent/env.example.json:3]: OPENAI_BASE_URL is probably a better name for this?
```suggestion
  "OPENAI_BASE_URL": "http://api.openai.com"
```

File: agent/ServerAgent.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ import env from "./env.json" assert { type: "json" };
 
 const configuration = new Configuration({
   apiKey: env.OPENAI_API_KEY,
+  basePath: `${env.OPENAI_PROXY_URL}/v1`
```
Comment (keldenl) [inline agent/ServerAgent.js:8]: same here
```suggestion
  basePath: `${env.OPENAI_BASE_URL}/v1`
```

**Review by keldenl — COMMENTED** [review]
small comment about variable naming

File: .env [hunk 3]
```diff
@@ -0,0 +1 @@
+PORT=3001
```
Comment (keldenl) [inline .env:1]: you also shouldn add the .env file into .gitignore, and add a new .env.template that people can clone

**Comment by PlexPt** [comment]
Done

**Comment by dzoba** [comment]
Sorry I'm not sure why we want this.  I'm going to close for now but if you'd like to open another PR with some more details on why we should have this, please feel welcome to do so.

**Pull request closed without merging**